### ISSUE TITLE
Implement Loss functions for Imagenet models

### ISSUE THREAD

**Issue by keisen** [keisen/tf-keras-vis#24]
We need some basic loss libraries to be ease to use.
